blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -115,7 +116,7 @@ const StoryCarousel = ({
 									maximum: elements.length,
 								})}`}
 							>
-								<a className={`${BLOCK_CLASS_NAME}__story-card`} href={url}>
+								<Link className={`${BLOCK_CLASS_NAME}__story-card`} href={url}>
```

Comment:
This removes the a11y violation

`<Link tabindex='-1' className={`${BLOCK_CLASS_NAME}__story-card`} href={url}>`

Reply:
Updated to use the `assistiveHidden` prop

Reply:
awesome!